lib/widgets/game.dart

```diff
@@ -39,6 +42,17 @@ class _GameState extends State<Game> {
 
   @override
   Widget build(BuildContext context) {
+    startTime = new DateTime.now();
```

Comment:
Game class is not used only for contest. It is used by student when playing alone also. Create your own ContestGame class. Then call Game from there. Do not change anything in Game.